docs/eventing/samples/kafka/resetoffset/README.md

```diff
@@ -1,91 +1,82 @@
-that they provide temporal persistence of events. This capability allows for
-certain advanced use cases to be supported, including the ability to "replay"
-prior events via repositioning the current location in the event-stream.  This
-capability can be extremely useful when attempting to recover from unexpected
-Subscriber downtime.
-
-The
-[ResetOffset CRD](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset)
-exposes the ability to manipulate the location of the ConsumerGroup Offsets in
-the event stream of a given Knative Subscription. Without the ResetOffset CRD
-you would be responsible for manually stopping ConsumerGroups and manipulating
-Offsets.
+# ResetOffset Example
 
-!!! note
-    Repositioning the ConsumerGroup Offsets will impact the event ordering and
-    is intended for failure recovery scenarios. This capability needs to be used
-    with caution only after reviewing the CRD documentation linked above.
+Kafka-backed Channels differ from other Channel implementations because they provide temporal persistence of events.
+
+Event persistence enables certain advanced use cases to be supported, such as the ability to replay prior events by repositioning their current location in the event stream. This feature is useful when attempting to recover from unexpected Subscriber downtime.
+
+The [ResetOffset custom resource definition (CRD)](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset) exposes the ability to manipulate the location of the ConsumerGroup Offsets in the event stream of a given Knative Subscription. Without the ResetOffset CRD, you must manually stop ConsumerGroups and manipulating Offsets.
 
 !!! note
     ResetOffsets are currently only supported by the Distributed KafkaChannel
-    implementation. It is expected that the Consolidated KafkaChannel
-    implementation will support them in the near future.
+    implementation.
 
-## Prerequisites
+!!! warning
+    Repositioning the ConsumerGroup Offsets will impact the event ordering and
+    is intended for failure recovery scenarios. This capability needs to be used
+    with caution only after reviewing the [CRD documentation](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset).
 
-- A Kubernetes cluster with a Knative
-  [Kafka Channel](https://knative.dev/docs/eventing/channels/channels-crds/)
-  installed.
-- A valid KafkaChannel and Subscription exist.
+## Prerequisites
 
-## Examples
+- A Kubernetes cluster with the [Kafka Channel implementation](https://knative.dev/docs/eventing/channels/channels-crds/) installed.
+- A valid KafkaChannel resource and Subscription exist.
 
-The following examples assume a Kubernetes Namespace called `my-namespace`
-containing a KafkaChannel Subscription named `my-subscription` and should be
-updated according to your use case.
+!!! note
+    The ResetOffset CRD is a single-use operation, and should be
```

Comment:
```suggestion
    The ResetOffset CRD is a single-use operation, and must be
```

Reply:
I actually asked the same thing, but this is a recommendation, not mandatory AFAIK, so I'll leave it as "should" for the moment, even though I know we'd prefer to not do that...